etc/spec/ARCHITECTURE.md

```diff
@@ -254,7 +277,20 @@ request, it gets created internally.
 
 ####Logging out
 
+This operation is performed by the client to the RESTful API server for logging out.
+
 A DELETE request on `/sessions`.
 
 During this request the server checks if this user has set a password. If not, then along 
 with the session, the user gets deleted, too.
+
+####Checking authentication
+
+This operation is performed by the real-time service to the RESTful API server for verifying
+that a given client cookie is correct.
+
+A GET request on `/sessions/<username>`. This request must be performed using the `ting_auth`
+cookie that the user provided. If the cookie is valid for the given user, a 204 (No Content) response
+is returned. Otherwise, a 403 (Forbidden) response is returned.
```

Comment:
Please follow the styling conventions from the `sessions` section on status codes for consistency. I didn't look at those portions when writing this, but we should aim to remain uniform. "the server returns a 204 status code" seems to be the language we're using there.